script-test/playbackstrategies/msestrategytest.js

```diff
@@ -1,491 +1,579 @@
-          setUpMSE();
-          mseStrategy.load('src', null, 15);
+          it('should initialise MediaPlayer with the expected parameters when startTime is set', function () {
+            setUpMSE();
+            mseStrategy.load('src', null, 15);
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=15', true);
-        });
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=15', true);
+          });
 
-        it('should initialise MediaPlayer with the expected parameters when startTime is set and there is a time correction', function () {
-          setUpMSE(1922);
-          mseStrategy.load('src', null, 15);
+          it('should initialise MediaPlayer with the expected parameters when startTime is set and there is a time correction', function () {
+            setUpMSE(1922);
+            mseStrategy.load('src', null, 15);
 
-          expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=1937', true);
-        });
+            expect(mockDashInstance.initialize).toHaveBeenCalledWith(mockVideoElement, 'src#t=1937', true);
+          });
 
-        it('should set up bindings to MediaPlayer Events correctly', function () {
-          setUpMSE();
-          mseStrategy.load(null, null, 0);
-
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('timeupdate', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('playing', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('pause', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('waiting', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('seeking', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('seeked', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('ended', jasmine.any(Function));
-          expect(mockVideoElement.addEventListener).toHaveBeenCalledWith('error', jasmine.any(Function));
-          expect(mockDashInstance.on).toHaveBeenCalledWith(dashjsMediaPlayerEvents.ERROR, jasmine.any(Function));
-          expect(mockDashInstance.on).toHaveBeenCalledWith(dashjsMediaPlayerEvents.MANIFEST_LOADED, jasmine.any(Function));
-          expect(mockDashInstance.on).toHaveBeenCalledWith(dashjsMediaPlayerEvents.MANIFEST_VALIDITY_CHANGED, jasmine.any(Function));
-          expect(mockDashInstance.on).toHaveBeenCalledWith(dashjsMediaPlayerEvents.QUALITY_CHANGE_RENDERED, jasmine.any(Function));
-          expect(mockDashInstance.on).toHaveBeenCalledWith(dashjsMediaPlayerEvents.METRIC_ADDED, jasmine.any(Function));
-        });
+          it('should attach a new audio source when CDN Failover occurs', function () {
+            setUpMSE(null, null, MediaKinds.AUDIO);
```

Comment:
I don't think this test is necessary as the exact same codepath is executed for both audio and video. The only variance is the first assertion should be `initialize` with an audio element, but this is already tested in the above new test.